### PR TITLE
Used templates, created basic layout and components as per design

### PR DESCRIPTION
**Pull request by ghost**
Used templates from Chakra UI, created basic layout and components as per design

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,18 +9,43 @@
...
     "eslint-config-next": "13.1.6",
+    "framer-motion": "^10.1.0",
     "next": "13.1.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "typescript": "4.9.5"
   },
   "devDependencies": {
+<<<<<<< Updated upstream
+=======
```
Comment (jeromevvb) [inline package.json]: @kevinjung13 make sure you get rid of your conflicted comments before you commit 
the package.json is not valid for this reason, the build failed on vercel

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,6 +1,49 @@
+<<<<<<< Updated upstream
```
Comment (jeromevvb) [inline src/pages/_app.tsx]: @kevinjung13 same here

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -1,6 +1,49 @@
...
+
+
+  return (
+    <ChakraProvider theme={theme}>
+      <Header />
+      <Component {...pageProps}/>
+    </ChakraProvider>
+  )
+>>>>>>> Stashed changes
 }
```
Comment (jeromevvb) [inline src/pages/_app.tsx:95]: @kevinjung13  this also

File: src/pages/create.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  Link
+} from '@chakra-ui/react';
+import { Children } from 'react';
+
+
+export default function Create() {
+  
+  const handleClick = () => {
+    <Button
+      isLoading
```
Comment (jeromevvb) [inline src/pages/create.tsx]: @kevinjung13 
You must use a state, `useState` 
`const [isLoading, setLoading] = useState(false)`
And then when you handle the click, you update the state  `setLoading(true)`

Your button below get the prop isLoading={isLoading}

File: src/pages/index.tsx [hunk 5]
```diff
@@ -118,6 +115,14 @@ export default function Home() {
...
         </div>
       </main>
+=======
+        <title>Oasis Marketplace</title>
+        <meta name='description' content='Oasis Marketplace: AI NFT Marketplace' />
+        <meta name='viewport' content='width=device-width, initial-scale=1' />
+        <link rel='icon' href='/logo.png'/>
+      </Head>
+      <Main />
+>>>>>>> Stashed changes
```
Comment (jeromevvb) [inline src/pages/index.tsx]: @kevinjung13 same here

File: src/styles/globals.css [hunk 6]
```diff
@@ -76,6 +77,8 @@
     --card-border-rgb: 200, 200, 200;
   }
 }
+=======
+>>>>>>> Stashed changes
```
Comment (jeromevvb) [inline src/styles/globals.css]: @kevinjung13 thoses comments here

**Review by jeromevvb — CHANGES_REQUESTED** [review]

File: src/components/Footer/icons.tsx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  _hover: object;
+}
+
+/* Social Media Buttons */
+export function SocialButton(props: SocialProps) {
+
+  const { children, label, href, _hover } = props;
+
+  return (
+    <chakra.button
```
Comment (jeromevvb) [inline src/components/Footer/icons.tsx]: @kevinjung13 
another approach can be to use the `Link` component from chakra use the props `isExternal` and `href`
This component should wrap the Button

File: src/components/Header/Devices/desktop.tsx [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+  return (
+    <>  
+      {/* Stacking */}
+      <Stack direction={'row'} spacing={4}>
+        {/* Options */}
+        {Links.map((link) => (
+        <Box key={link.name}>
+          <Popover trigger={'hover'} placement={'bottom-start'} >
+            <PopoverTrigger>
+              <Link
```
Comment (jeromevvb) [inline src/components/Header/Devices/desktop.tsx]: If you're using NextJS you should use `next/link` component to navigate through the app
If you a normal `<a>` you force your all app to refresh
try it and see the difference

https://nextjs.org/docs/api-reference/next/link

File: src/components/Header/index.tsx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+    href: "/create"
+  },
+
+  {
+    name: "Explore",
+    href: "/explore"
+  },
+
+  {
+    name: "Connect Wallet",
```
Comment (jeromevvb) [inline src/components/Header/index.tsx:31]: @kevinjung13 good idea but connect wallet we will replace with https://www.rainbowkit.com/
do you feel capable to integrate it?

File: src/components/Header/logo.tsx [hunk 10]
```diff
@@ -0,0 +1,20 @@
+import { HStack, Link, Image } from "@chakra-ui/react";
+
+export default function Logo() {
+  return (
+    <>
+      {/* Stacking */}
+      <HStack spacing={8} align={'center'}>
+        {/* Logo */}
+        <Link href={'/main'} _hover={{ opacity: '50%' }}>
```
Comment (jeromevvb) [inline src/components/Header/logo.tsx]: @kevinjung13 main does not exist should be `/` for index
Same here, make sure to use next/link

File: src/pages/Create/index.tsx [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+import { useState } from 'react';
+
+export default function Create() {
+
+    /* Loading State for button */
+  const [isLoading, setIsLoading] = useState(false)
+  
+    /* HandleClick function */
+  const handleClick = () => {
+    setIsLoading(status => !status)
```
Comment (jeromevvb) [inline src/pages/Create/index.tsx]: @kevinjung13 I need the value containing in the field here ;)

File: src/pages/Create/index.tsx [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+            textAlign={'center'}>
+            Create Your NFT 
+          </Heading>
+          {/* Form */}
+          <Form />
+          {/* Stacking of Buttons */}
+          <Stack spacing={6}>
+            <Button
+              as={'a'}
+              href={'/create/results'}
```
Comment (jeromevvb) [inline src/pages/Create/index.tsx]: Why are going to a new page? We can handle the request in the same page no?
How would you change your approach?

File: src/pages/_app.tsx [hunk 13]
```diff
@@ -1,6 +1,59 @@
...
+/* Dosis fonts */
+    
+import '@fontsource/dosis';
+import '@fontsource/dosis/700.css';
+import '@fontsource/dosis/500.css';
+
+/* Themes */
+
+const theme = extendTheme({
+  colors: {
```
Comment (jeromevvb) [inline src/pages/_app.tsx:18]: @kevinjung13 check the colors system from chakra https://chakra-ui.com/docs/styled-system/theme
we should leverage the colorScheme feature of chakra and provide the full color palette

**Review by jeromevvb — CHANGES_REQUESTED** [review]

File: src/components/Footer/icons.tsx (hunk 7 above)
Comment (ghost) [reply src/components/Footer/icons.tsx]: @jeromevvb Ah perfect, just tried and it works well so ill do that. Thanks!

File: src/components/Header/logo.tsx (hunk 10 above)
Comment (ghost) [reply src/components/Header/logo.tsx]: Done, and I now understand the difference next/link makes. Thanks for sharing!

File: src/components/Header/Devices/desktop.tsx (hunk 8 above)
Comment (ghost) [reply src/components/Header/Devices/desktop.tsx]: Done!

**Pull request closed without merging**